compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/viewinterop/InteropView.uikit.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.viewinterop
+
+import androidx.compose.ui.interop.InteropWrappingView
+import kotlinx.cinterop.CValue
+import platform.CoreGraphics.CGPoint
+import platform.UIKit.NSStringFromCGPoint
+import platform.UIKit.UIEvent
+import platform.UIKit.UIView
+
+/**
+ * A typealias for the platform's built-in View type, which may be hosted inside of a Compose UI
+ * hierarchy to allow for interoperability. Not all platforms support interoperability in this way,
+ * in which case the typealias will resolve to [Any].
+ */
+actual class InteropView internal constructor (
```

Comment:
Let's typealias this to platform `UIView` and avoid wrapper around wrapper

Reply:
Done